### DISCUSSION TITLE
Is Github hiring remote only devs?

### DISCUSSION THREAD

**Discussion by IonelLupu** [community/community#139873] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

We live in a world where everyone is everywhere and we can stay connected at all times.

Having this said, is Github hiring remote devs around the world? If so, can someone help me get in touch with a recruiter from Github? I am a full-stack developer working with React/Angular and Node.js with over 10years of experience

**Comment by ebndev** [comment, marked as answer]
Hi @IonelLupu, thanks for being a part of the GitHub Community!

Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. 

However, to answer your question - yes we are a remote first company. You can view openings on our [career page](https://www.github.careers/careers-home). Best of luck!

**Reply by IonelLupu** [reply]
@ebndev thank you for answering the question and not just close the discussion. I didn't know where to ask this to be honest. 

I was a bit thrown off by the fact it says "US-remote" implying you are hiring only devs from US and not world wide. 😅 
![image](https://github.com/user-attachments/assets/411ae623-ee41-4560-b93a-582daccb2f5b)

**Reply by ebndev** [reply]
I can see why that may be confusing! As you see on the page, the roles do list where the team is hiring from. So although we do hire globally, it's not applicable for every role.
